catalog.json

```diff
@@ -1,18 +1,18 @@
 {
   "id": "8478b533-2b59-4007-8494-2feec5970f94",
   "name": "cdn-route",
-  "description": "Custom URL, CDN, and free TLS certificates",
+  "description": "Custom domain, CDN caching, and TLS certificates",
   "bindable": true,
   "metadata": {
-    "displayName": "Content Distribution Network",
-    "documentationUrl": "https://cloud.gov/docs/services/cdn-route"
+    "displayName": "CDN Route",
```

Comment:
It just seems a little confusing only call it "Content Distribution Network", since it does more than just CDN service, but happy for a correction on that!